platform/install/gitops.mdx

```diff
@@ -126,8 +126,6 @@ EOF
 
 <!-- vale on -->
 
-<ListHelmVersions />
```

Comment:
Can you explain why this is removed?

Reply:
I think the helm repo search was misplaced here in the [ArgoCD Section](https://www.vcluster.com/docs/platform/install/gitops#apply-argocd-application). The correct place is [here](https://www.vcluster.com/docs/platform/install/helm#deploying-the-platform)

![image](https://github.com/user-attachments/assets/9a568e69-ac9b-4f10-868f-822518614b48)

Reply:
This should be in both places, helm and argo install.